src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,47 @@
-export const Tabs = () => {};
+import '@fortawesome/fontawesome-free/css/all.css';
+import React, { useState } from 'react';
+
+export const Tabs = ({
+  tabs,
+  activeTabId,
+  onTabSelected,
+  onTitle,
+  onContent,
+}) => {
+  const [content, setContent] = useState('Some text 1');
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(element => (
+            <li
+              className={element.id === activeTabId ? 'is-active' : ''}
+              data-cy="Tab"
+              key={element.id}
+            >
+              <a
+                href={`#${element.id}`}
+                data-cy="TabLink"
+                id={element.id}
+                onClick={event => {
+                  if (event.target.className !== 'is-active') {
+                    onTabSelected(event.target.id);
+                    onTitle(element.title);
+                    setContent(element.content);
+                  }
+                }}
+              >
```

Comment:
try to avoid such comparison with DOM element's property. use id for comparison
and move this logic to handler

And what does onTitle do? you don't pass such prop